hybridse/src/udf/udf.cc

```diff
@@ -653,33 +654,24 @@ void string_to_int(StringRef *str, int32_t *out, bool *is_null_ptr) {
     if (0 == str->size_) {
         return;
     }
-    try {
-        // string -> integer
-        // std::string::size_type sz;  // alias of size_t
-        // *out = std::stoi(str->ToString(), &sz);
-        // if (sz < str->size_) {
-        //    *out = 0;
-        //    *is_null_ptr = true;
-        //    return;
-        //}
-        std::string str_obj = str->ToString();
-        const char *c_str = str_obj.c_str();
-        char *end;
-        *out = strtol(c_str, &end, 10);
-        if (end < c_str + str->size_) {
-            *out = 0;
-            *is_null_ptr = true;
-            return;
-        }
-        *is_null_ptr = false;
-    } catch (...) {
-        // error management
+    std::string str_obj = str->ToString();
+    const char *c_str = str_obj.c_str();
+    char *end;
+    *out = strtol(c_str, &end, 10);
+    if (end < c_str + str->size_) {
+        *out = 0;
+        *is_null_ptr = true;
+        return;
+    }
+    if (errno == ERANGE) {
```

Comment:
detect error for `strtoXX` is complex, you can see 
- https://stackoverflow.com/a/26083517/6187385

To be simple, try `absl::SimpleAtoi`